### DISCUSSION TITLE
Custom Sounds

### DISCUSSION THREAD

**Discussion by darkerheart** [tom-james-watson/breaktimer-app#164] [Ideas]
Hi, first I'd very much like to share my tremendous appreciation for the app!!!

If there are any further updates made to this app - I would like to suggest the ability to choose your own custom sound (instead of the gong sound). Over time I find that the app becomes less effective as my brain tries to learn to ignore the break timer prompt. I that I have become desensitised to the gong sound, sometimes even ignoring it completely without realising. The ability to switch out sounds every now and then would really help to resensitise myself to the app, and reduce my ability to ignore it subconsciously!

Also are there any plans to make something like this into an iPhone app? I would be willing to pay for it!

Thank you

**Comment by puffer3** [comment]
I too want custom sounds very much! The gong noise has a high noise floor on my fancy work speakers. Thanks!

**Comment by tom-james-watson** [comment]
Hey if you have good sounds, I'd be open to adding them to the app. Would be easier than custom sounds.

**Reply by echo-64** [reply]
I’m not an app developer, so I don't know the technical difficulties behind it. But it would be better to give users the option to upload their own, rather than adding every single sound someone suggests. The feature name is 'custom sounds', not 'suggested'. Thanks.
